### PR TITLE
BC-740 remove backwards compatibility for ansible image tag variable

### PR DESCRIPTION
**Pull request by alriese**
# Description

https://ticketsystem.hpi-schul-cloud.org/browse/BC-864

## Approval for review

- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Review by mamutmk5 — CHANGES_REQUESTED** [review]
No, thats not the right time for it to remove it.
Please lock at schulcloud-editor and edtrio who is includet at the ansible playbock https://github.com/hpi-schul-cloud/dof_app_deploy/blob/main/ansible/playbook.yml#L25-L26 .

https://github.com/hpi-schul-cloud/schulcloud-editor
https://github.com/hpi-schul-cloud/edtrio

**Comment by virgilchiriac** [comment]
> No, thats not the right time for it to remove it. Please lock at schulcloud-editor and edtrio who is includet at the ansible playbock https://github.com/hpi-schul-cloud/dof_app_deploy/blob/main/ansible/playbook.yml#L25-L26 .
> 
> https://github.com/hpi-schul-cloud/schulcloud-editor https://github.com/hpi-schul-cloud/edtrio

let's change that please - even if repo is archived, it is in use (only repos not in use should be archived)

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -48,15 +48,12 @@ jobs:
       - working-directory: ${{github.workspace }}/${{ matrix.repos }}
         shell: bash
         run: |
-          commit_id=$(git rev-parse HEAD)
-          mkdir -pv ansible/group_vars/all
```
Comment (virgilchiriac) [inline .github/workflows/deploy.yml:52]: is this really not needed?

**Comment by alriese** [comment]
Will be handled later

**Pull request closed without merging**